# Permutations

## Problem-6-64

Evaluate (a) ${}_4 \mathrm{ P }_2$, (b) ${}_7 \mathrm{ P }_5$, and (c) ${}_{10} \mathrm{ P }_3$.

In [2]:
cat( "(a) ", choose(4,2)*factorial(2), "\n")
cat( "(b) ", choose(7,5)*factorial(5), "\n")
cat( "(c) ", choose(10,3)*factorial(3), "\n")

(a)  12 
(b)  2520 
(c)  720 


---

## Problem-6-65
For what value of $n$ is ${}_{n+1} \mathrm{ P }_3 = {}_n \mathrm{ P }_4$?<br>

\begin{align*}
{}_{n+1} \mathrm{ P }_{3} &= (n + 1)(n)(n - 1) \\
{}_{n} \mathrm{ P }_{4}   &= (n)(n - 1)(n - 2)(n - 3) \\ \\ \\
(n + 1) &= (n - 2)(n - 3) \\
0 &= n^2 - 6n + 5 \\ \\
n &= 5 \\
n &= 1
\end{align*}

In this context the answer that satisfies the equation is $5$.

---

## Problem-6-66

In how many ways can $5$ people be seated on a sofa if there are only $3$ seats available?

\begin{align*}
5 \cdot 4 \cdot 3 = 60
\end{align*}

In [4]:
choose(5,3) * factorial(3)

[1] 60

---

## Problem-6-67

In how many ways can 7 books be arranged on a shelf if (a) any arrangement is possible, (b) 3 particular books must always stand together, and (c) 2 particular books must occupy the ends?

### Solution-6-67

In [32]:
cat("(a) Any arrangement: ", factorial(7), "\n")
cat("(b) If 3 Particular books must always stand together: ", factorial(3)*factorial(5), "\n")
cat("(c) 2 books must occupy the ends: ", factorial(5)*2, "\n")

(a) Any arrangement:  5040 
(b) If 3 Particular books must always stand together:  720 
(c) 2 books must occupy the ends:  240 


Let's solve each part of the problem separately:

(a) Any arrangement is possible:
In this case, there are 7 books to arrange on the shelf, and for each book, there are 7 possible positions. Therefore, the total number of arrangements is $7!$, which is $7 \times 6 \times 5 \times 4 \times 3 \times 2 \times 1 = 5040$.

(b) 3 particular books must always stand together:
Treat the 3 particular books as a single entity. Now, we have 5 entities (4 single books and the group of 3 books) to arrange on the shelf. Within the group of 3 particular books, there are $3!$ ways to arrange them. Thus, the total number of arrangements is $5! \times 3!$, which is $5 \times 4 \times 3 \times 2 \times 1 \times 3 \times 2 \times 1 = 720$.

(c) 2 particular books must occupy the ends:
Consider the 2 particular books as fixed entities. Now, we have 5 remaining books to arrange in the middle. There are $5!$ ways to arrange these books. Additionally, the 2 particular books can be arranged in 2 ways (either they can be at the left end or the right end). Therefore, the total number of arrangements is $5! \times 2$, which is $5 \times 4 \times 3 \times 2 \times 1 \times 2 = 240$.

So, the number of ways to arrange the books on the shelf are:<br>
(a) 5040 ways<br>
(b) 720 ways<br>
(c) 240 ways

## Problem-6-68

How many numbers consisting of five different digits each can be made from the digits $1, 2, 3, \ldots, 9$ if (a) the numbers must be odd and (b) the first two digits of each number are even?

### Solution-6-68

Let's solve each part of the problem separately:

(a) Numbers must be odd:<br>
For a number to be odd, the units digit must be an odd number. We have 5 odd numbers $1, 3, 5, 7, 9$ to choose from for the units place. After choosing the units digit, there are 8 remaining digits (including 0) for the tens, hundreds, thousands, and ten-thousands places. Since the digits must be different, we have 8 choices for the tens digit, 7 choices for the hundreds digit, 6 choices for the thousands digit, and 5 choices for the ten-thousands digit. Therefore, the total number of numbers is $5 \times 8 \times 7 \times 6 \times 5 = 8400$.

(b) The first two digits of each number are even:<br>
For the first two digits to be even, we have 4 even numbers $2, 4, 6, 8$ to choose from for the tens and units places. After choosing the first two digits, there are 7 remaining digits (including 0) for the hundreds, thousands, and ten-thousands places. Similarly, we have 7 choices for the hundreds digit, 6 choices for the thousands digit, and 5 choices for the ten-thousands digit. Therefore, the total number of numbers is $4 \times 7 \times 6 \times 5 = 840$.

So, the number of numbers satisfying the given conditions are:<br>
(a) 8400 numbers<br>
(b) 840 numbers

---

## Problem-6-69

Solve Problem 6.68 if repetitions of the digits are allowed.

### Solution-6-69

If repetitions of the digits are allowed, the approach to solving the problem changes slightly.

(a) Numbers must be odd:<br>
For a number to be odd, the units digit must be an odd number. We have 5 odd numbers $1, 3, 5, 7, 9$ to choose from for the units place. For the tens, hundreds, thousands, and ten-thousands places, we have all 9 digits (1 through 9) available, including the even numbers. Therefore, for each of these places, we have 9 choices. Hence, the total number of numbers is $5 \times 9 \times 9 \times 9 \times 9 = 32805$.

(b) The first two digits of each number are even:<br>
For the first two digits to be even, we have 4 even numbers $2, 4, 6, 8$ to choose from for the tens and units places. For the remaining three places (hundreds, thousands, and ten-thousands), we have all 9 digits (1 through 9) available, including the even numbers. Therefore, for each of these places, we have 9 choices. Hence, the total number of numbers is $4 \times 9 \times 9 \times 9 \times 9 = 26244$.

So, the number of numbers satisfying the given conditions are:<br>
(a) 32805 numbers<br>
(b) 26244 numbers

---

## Problem-6-70

How many different three-digit numbers can be made with three 4's, four 2's, and two 3's?

### Solution-6-70

To find the number of different three-digit numbers that can be made with three 4's, four 2's, and two 3's, we'll use permutations with repetition.

The total number of digits we have is $3 + 4 + 2 = 9$.

Using permutations with repetition, the number of ways to arrange these digits in a three-digit number is:

$P(9, 3) = \frac{9!}{(9-3)!} = \frac{9!}{6!} = \frac{9 \times 8 \times 7 \times 6!}{6!} = 9 \times 8 \times 7 = 504$

However, we must account for the repetition of digits. 

- There are $3!$ ways to arrange the three 4's within each arrangement. 
- There are $4!$ ways to arrange the four 2's within each arrangement. 
- There are $2!$ ways to arrange the two 3's within each arrangement. 

So, the total number of different three-digit numbers that can be made is:

$\frac{504}{3! \times 4! \times 2!} = \frac{504}{6 \times 24 \times 2} = \frac{504}{288} = 21$

Therefore, there are 21 different three-digit numbers that can be made with three 4's, four 2's, and two 3's.

---

## Problem-6-71

In how many ways can 3 men and 3 women be seated at a round table if (a) no restriction is imposed, (b) 2 particular women must not sit together, and (c) each woman is to be between 2 men.

Let's solve each part of the problem separately:

(a) No restriction is imposed:<br>
In this case, we consider the circular arrangement of 3 men and 3 women as a permutation. The number of ways to arrange 6 people around a circular table is $(6 - 1)! = 5!$. Therefore, there are $5! = 120$ ways to seat 3 men and 3 women at a round table.

(b) 2 particular women must not sit together:<br>
If two particular women cannot sit together, we treat them as one entity. So, we have 4 entities to arrange (1 group of 2 women, 3 men). Within the group of 2 women, there are $2!$ ways to arrange them. Therefore, the number of arrangements is $4! \times 2! = 24 \times 2 = 48$.

(c) Each woman is to be between 2 men:<br>
In this case, we fix one man at the top of the table (arbitrarily). Now, we have 3 men and 3 women to arrange alternatively. We can arrange 3 men in $3!$ ways and 3 women in $3!$ ways. So, the total number of arrangements is $3! \times 3! = 6 \times 6 = 36$.

So, the number of ways to seat 3 men and 3 women at a round table are:<br>
(a) 120 ways<br>
(b) 48 ways<br>
(c) 36 ways

The reason why we use $(6 - 1)!$ to calculate the number of ways to arrange 6 people around a circular table is due to the circular nature of the arrangement.

When arranging people around a circular table, we fix one person's position as a reference point. Then, we arrange the remaining $n - 1$ people relative to this fixed position.

Consider a linear arrangement first. If we were arranging $n$ people in a row (a linear arrangement), we would have $n!$ ways to arrange them because each person has $n$ possible positions to occupy.

Now, when we arrange people around a circular table, we still have $n$ people to arrange, but the arrangement is circular. This means that once we fix one person's position, the arrangement becomes linear relative to that fixed person. Since the circular arrangement can be rotated without changing its nature, there are $n$ different starting positions around the table. Therefore, we divide $n!$ by $n$ to account for the circular symmetry, resulting in $(n - 1)!$ ways to arrange \(n\) people around a circular table.

In the case of arranging 6 people around a circular table, we have $6 - 1 = 5$ possible starting positions around the table. Hence, the number of ways to arrange them is $(6 - 1)! = 5!$.